cmd/aws-sigv4-proxy/main.go

```diff
@@ -119,6 +142,7 @@ func main() {
 		},
 	}
 
+	log.WithFields(log.Fields{"Custom Headers": *customHeaders}).Infof("Custom headers: [%s]", *customHeaders)
```

Comment:
Could there be a situation where you want to avoid logging a sensitive header value? In this case, I suppose the solution could be just to log the key?